src/main/drivers/vtx_soft_spi_rtc6705.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "platform.h"
+
+#ifdef USE_RTC6705
+
+#include "drivers/bus_spi.h"
+#include "drivers/system.h"
+#include "drivers/light_led.h"
+
+#include "vtx_soft_spi_rtc6705.h"
+
+#define RTC6705_SPICLK_ON     IOHi(rtc6705ClkPin)
+#define RTC6705_SPICLK_OFF    IOLo(rtc6705ClkPin)
+
+#define RTC6705_SPIDATA_ON    IOHi(rtc6705DataPin)
+#define RTC6705_SPIDATA_OFF   IOLo(rtc6705DataPin)
+
+#define RTC6705_SPILE_ON      IOHi(rtc6705LePin)
+#define RTC6705_SPILE_OFF     IOLo(rtc6705LePin)
+
+char *vtx_bands[] = {
+    "BOSCAM A",
+    "BOSCAM B",
+    "BOSCAM E",
+    "FATSHARK",
+    "RACEBAND",
+};
```

Comment:
The text for `vtx_bands` should not be defined in the device driver. This should be in the 'io/' layer.